public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
That's fine. What I meant was why not use the same visibility status as the parent container? Change the original html code of sound and mute button to have the same visibility as others in the container?

> it uses the visible class because the mute button uses the hidden class (they switch off).
Didn't quite understand what you meant here.